README.md

```diff
@@ -1 +1,37 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# Self Introduction
+Hello. My name is __Soomin Kim__. I am a MA student of _Committee on International Relations_. I was born and raised in South Korea for 23 years and came to the United States to continue my study. I have one younger sister, who is also studying in the Unites States. We love and take care of each other for we are the only family members outside of South Korea. 
```

Comment:
Minor: spell check Unites States -> United States

Reply:
Not that it would help in this instance since "unites" is also a word, but RStudio includes a built-in spell checker. Handy for Markdown/RMarkdown documents